packages/cli/src/cmds/weakSubjectivityState.ts

```diff
@@ -4,5 +4,5 @@ export enum WeakSubjectivityServers {
   prater = "http://localhost:8081",
   pyrmont = "http://localhost:8081",
   dev = "http://localhost:8081",
-  coorparoo = "http://localhost:8081",
```

Comment:
What are these servers / services?

Reply:
These would be addresses for servers that would be running this code (assuming we decide to go with this scheme):
https://github.com/ChainSafe/eth2-ws-provider/pull/1